KubernetesTest.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
just checking that we'll be removing this in favor of unit tests when we merge, correct? we should avoid using ipynbs for tests, and theyre also quite large/slow to send to the cluster which is done by default

Reply:
in terms of examples, we can also add some more info to the core [cluster API docs](https://github.com/run-house/runhouse/blob/main/docs/api/python/cluster.rst)

Reply:
yup, I think it makes sense to remove the notebook and split the content out into
* unit tests in tests/ folder
* examples in rh docs, either within docstrings or explicit k8s section
* tutorial for k8s cluster. the compute tutorial under clusters probably makes the most sense for now